TriggerMode: 0
TriggerSource: 0
SysteamTime: false